packages/web-react/src/components/Tabs/TabLink.tsx

```diff
@@ -1,17 +1,20 @@
 import React from 'react';
+import classNames from 'classnames';
 import { ChildrenProps, TransferProps } from '../../types';
 import { useTabsStyleProps } from './useTabsStyleProps';
+import { useStyleProps } from '../../hooks';
 
 export interface TabLinkProps extends ChildrenProps, TransferProps {
   href: string;
 }
 
 const TabLink = ({ children, href, ...restProps }: TabLinkProps): JSX.Element => {
   const { classProps } = useTabsStyleProps();
+  const { styleProps, props: transferProps } = useStyleProps(restProps);
 
   return (
-    <li className={classProps.item}>
-      <a {...restProps} href={href} className={classProps.link} role="tab">
+    <li {...transferProps} className={classNames(classProps.item, styleProps.className)}>
```

Comment:
I think you should update the API table in readme as well.

Reply:
❤️

Reply:
Isn't this a BC? Because if someone used restprops to pass something to the <a>, it is not passed to the <li> instead? Or am I reading this wrong?

Reply:
Changed back and added `itemProps` to `<TabLink />` that will be used in parent `<li>` element